src/ledger/LedgerTxn.cpp

```diff
@@ -825,6 +825,35 @@ LedgerTxn::Impl::erase(InternalLedgerKey const& key)
     }
 }
 
+void
+LedgerTxn::addRestoredFromHotArchive(LedgerEntry const& ledgerEntry, LedgerEntry const& ttlEntry)
```

Comment:
@SirTyson `restoreFromHotArchive` `restoreFromLiveBucketList` can be removed right? If we remove the v23 code from `ApplyHelper`, these will no longer be used.

Reply:
Looks like the restore op uses these methods pre-v23.